### ISSUE TITLE
deprecate?

### ISSUE THREAD

**Issue by jimmywarting** [lazd/gulp-declare#13]
Deprecated npm package and archive the repo? No update in years

**Comment by lazd** [comment]
Yeah I haven't worked on this project in forever, but there don't seem to activity in the existing issues, it has 8k downloads every week, and it still works.
